pandas/tests/test_nanops.py

```diff
@@ -548,6 +548,17 @@ def test_nancorr_spearman(self):
         self.check_nancorr_nancov_1d(nanops.nancorr, targ0, targ1,
                                      method='spearman')
 
+    def test_nancorr_bicor(self):
```

Comment:
These checks are great, but I don't think anything is verifying that you actually implemented the formula correctly. Can you add a few toy cases with actual numbers that would distinguish `bicor` from the other methods?